resources/assets/tconstruct/lang/fr_FR.lang

```diff
@@ -862,8 +938,7 @@ option.tconstruct.tank=Réservoir
 option.tconstruct.casting=Moulage
 option.tconstruct.progress=Progression de la recette
 
+
 # Config
-config.synced.restart=Votre configuration a été mise à jour pour correspondre au serveur. Relancez Minecraft.
+config.synced.restart=votre configuration a été mise à jour pour correspondre au serveur. Relancez Minecraft.
```

Comment:
Why is this decapitalized?  This is the message the game gives in chat when the configs sync on server join.